corehq/apps/reports/tasks.py

```diff
@@ -381,15 +381,18 @@ def find_question_id(form, value):
 
     for k, v in form.items():
         if isinstance(v, dict):
-            ret = find_question_id(v, value)
+            ret = find_question_id(v, value, last_path=last_path)
             if ret:
                 return [k] + ret
         elif isinstance(v, list):
             for repeat in v:
-                ret = find_question_id(repeat, value)
+                ret = find_question_id(repeat, value, last_path=last_path)
                 if ret:
                     return [k] + ret
         else:
+            if last_path:
+                v = os.path.basename(os.path.normpath(v))
```

Comment:
Curious if `normpath` was added here for a specific reason, or if it was added more because it seemed like a good idea? If the former, is there a test that will fail if it's not used? If the latter, I'm not sure it's necessary and it may be safer not to use it.

Also might be worth making this conditional on whether the path is absolute. Something like
```suggestion
            if last_path and os.path.isabs(v):
                v = os.path.basename(os.path.normpath(v))
```

Reply:
Alternately, maybe do this check outside the call, and don't even call `find_question_id(..., last_path=True)` if the path is not absolute?

Reply:
Unfortunately the check has to be done inside the call since it is done on `v` not the passed in parameter.